src/java.base/aix/classes/jdk/internal/loader/ClassLoaderHelper.java

```diff
@@ -22,10 +22,9 @@
  * or visit www.oracle.com if you need additional information or have any
  * questions.
  */
-
```

Comment:
Please keep this blank line (consistent with other `.java` source files).

Reply:
Restored it.